src/components.ts

```diff
@@ -152,6 +160,10 @@ export async function initComponents(): Promise<AppComponents> {
     pubsub,
     analytics
   })
+
+  const slackToken = await config.requireString('SLACK_BOT_TOKEN')
```

Comment:
Shall we create a fallback/mock component to use whenever `SLACK_BOT_TOKEN` isn't provided? (most likely to happen when running locally)

Reply:
I only moved this component higher up in the code. The token is included in the .env.default file, so the require statement won't fail. However, you're correct that if the flow you're testing locally needs to send messages to Slack, it will eventually encounter an error. 

Perhaps we can address this in a separate pull request.